packages/cli/src/lib/snapshot/reportPreviewer/reportPreviewer.ts

```diff
@@ -102,8 +157,6 @@ export class ReportViewer {
   }
 
   private logResourceErrors(ResourceSnapshotType: string) {
```

Comment:
`logResourceErrors` (as of de55db1) is only refactors, fret not.

Reply:
well, 494180c changed that:
We were parsing twice the report, once to get the error, once to get the missing vault entries.
So instead, during the parsing/creation of the snapshotReporter, we create the data structure for the reportPreviewer.

Note that the `snapshot` was taken prior to this big change. So, the fact that they work means that the refactoring didn't break everything (famouslastwords, ofc, the E2E hasn't been thru... yet)

Reply:
(note, sha has changed, 'cause rebase)